pkg/model/label.go

```diff
@@ -21,6 +21,10 @@ func GetArchivePathPrefixToLabels(repo string) string {
 	return fmt.Sprint(getArchivePathToLabels(), repo+"/")
 }
 
+func GetArchivePathPrefixToLabelPrefix(repo string, prefix string) string {
```

Comment:
i agree with this sentiment based on python, lisp, etc. and disagree based on C, and so on.

overall, there is an expectation of some mild namespace crowding since [ominous music] golang doesn't have templates/generics.

overall, varadic functions might as well be used where applicable, i agree.